Remotion/Web/ClientScript/tsconfig/release/tsconfig.json

```
@@ -1,7 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "../../bin/Release/dist",
+    "outFile": "../../bin/Release/dist/Web.ClientScript.js",
```

```suggestion
    "outFile": "../../bin/Release/dist/Remotion.Web.ClientScript.js",
```
alternative: just "ClientScript.js"